flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/MergingSharedSlotProfileRetrieverFactory.java

```diff
@@ -41,16 +41,12 @@
 class MergingSharedSlotProfileRetrieverFactory implements SharedSlotProfileRetriever.SharedSlotProfileRetrieverFactory {
 	private final PreferredLocationsRetriever preferredLocationsRetriever;
 
-	private final Function<ExecutionVertexID, ResourceProfile> resourceProfileRetriever;
-
 	private final Function<ExecutionVertexID, AllocationID> priorAllocationIdRetriever;
 
 	MergingSharedSlotProfileRetrieverFactory(
```

Comment:
Seems extra indentation of all method params in this class are removed unexpectedly.